### ISSUE TITLE
Consolidate TrackSerializers

### ISSUE THREAD

**Issue by jamesjwarren** [thisissoon/FM-API#47]
There are currently two track serializers one in  `fm.thirdparty.spotify` and the other in `fm.serializers.spotify`. Could these be consolidated?

**Comment by radeklos** [comment]
I agree that those serialisers are almost same. But serialisers from `fm.thirdparty.spotify` are used to load json from the spotify api to objects and  `fm.serializers.spotify` are used with soon fm database objets. I would rather split up spotify and soon fm models into two different layers.

**Comment by krak3n** [comment]
The problem was when both serialisers were required in the same module we would have a naming conflict we could get around it with the `as` notation but meh I'm not a huge fan of that.

It was a little confusing what the purpose of `fm.thirdparty.spotify` vs `fm.serializers.spotify` was. Could you improve the docstring strings to make it clear when and where they should be used as an interim? I'm also not sure the `thridparty` module name describes properly what the code inside this module is for, perhaps we should have:

```
external
- spotify
  - serializers.py
- echonest
  - serializers.py
```

Dunno thats just of the top of my head.

Generally I think things are starting to get a little bit unorganised, maybe we should wrap our API serving stuff in a more explicit `api` module, anyway lets chat about it later @radeklos / @jamesjwarren / @peeklondon?
